src/SelfServe/SelfServeTypes.ts

```diff
@@ -236,6 +236,7 @@ export enum DescriptionType {
 export interface Description {
   /**
    * Key used to pickup the string corresponding to the text to be shown as part of the description, from the strings JSON file.
+   * In case key is not present in strings JSON file, displays key as is.
```

Comment:
Can you remove this. This is not specific to the Description, it is for any translation key. I will add it elsewhere in the documentation